sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringOperations.scala

```diff
@@ -597,6 +597,23 @@ case class FormatString(children: Expression*) extends Expression with ImplicitC
 }
 
 /**
+ * Returns string, with the first letter of each word in uppercase,
+ * all other letters in lowercase. Words are delimited by whitespace.
```

Comment:
since you need to fix python, please also update documentation here to "remove all other letters in lowercase".